jvb/src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -418,8 +415,8 @@ public JSONObject getDebugState()
         debugState.put("effectiveVideoConstraints", effectiveConstraintsMap);
         debugState.put("lastN", lastN);
         debugState.put("supportsRtx", supportsRtx);
-        debugState.put("oversending", oversending);
-        debugState.put("total_oversending_time_secs", oversendingTimeTracker.totalOversendingTime().getSeconds());
+        debugState.put("oversending", oversendingTimeTracker);
```

Comment:
Don't think this is going to render anything useful.

Reply:
It got caught by unit tests :)

Reply:
👍  I got saved by those recently too